docs/source/Datasets.rst

```diff
@@ -0,0 +1,147 @@
+Datasets
+========
+.. currentmodule:: hub
+
+Creating Datasets
+~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    hub.dataset
+    hub.empty
+    hub.like
+    hub.ingest
+    hub.ingest_kaggle
+    hub.ingest_dataframe
+    hub.ingest_huggingface
+
+Loading Datasets
+~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    hub.load
+
+Deleting and Renaming Datasets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    hub.delete
+    hub.rename
+
+Copying Datasets
+~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+    
+    hub.copy
+    hub.deepcopy
+
+.. currentmodule:: hub.core.dataset
+
+Dataset Operations
+~~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    Dataset.summary
+    Dataset.append
+    Dataset.extend
+    Dataset.copy
+    Dataset.delete
+    Dataset.rename
+    Dataset.visualize
+    Dataset.pop
+    Dataset.rechunk
+    Dataset.flush
+    Dataset.clear_cache
+    Dataset.size_approx
+
+Dataset Visualization
+~~~~~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    Dataset.visualize
+    
+Dataset Credentials
+~~~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    Dataset.add_creds_key
+    Dataset.populate_creds
+    Dataset.update_creds_key
+    Dataset.change_creds_management
+    Dataset.get_creds_keys
+
+Dataset Properties
+~~~~~~~~~~~~~~~~~~
+.. autosummary::
+    :toctree:
+    :nosignatures:
+
+    Dataset.tensors
+    Dataset.groups
+    Dataset.read_only
+    Dataset.token
+    Dataset.info
```

Comment:
I know they are in vc section but branches, commits, commit_id can be here as well

Reply:
Not sure if we want multiple links to same stuff in the same page. Would it be better to move version control to above Dataset Views? That way it'll be right under properties

Reply:
Moved Version Control to just below Properties